src/geocoders/here.ts

```diff
@@ -82,10 +86,97 @@ export class HERE implements IGeocoder {
   }
 }
 
+export class HEREv2 implements IGeocoder {
+
+  options: HereOptions = {
+    serviceUrl: 'https://geocode.search.hereapi.com/v1/discover',
+    reverseGeocodeUrl: 'https://revgeocode.search.hereapi.com/v1/revgeocode',
```

Comment:
Please use `serviceUrl: 'https://geocode.search.hereapi.com/v1'` and append `/discover` or `/revgeocode` for the individual requests. Almost all other geocoders only take a single `serviceUrl` option.